node/third_party/openai-gpt-oss/metal/scripts/create-local-model.py

```
@@ -0,0 +1,358 @@
+        os.path.join(options.src, fname)
+        for fname in os.listdir(options.src)
+        if fname.endswith(".safetensors")
+    ]
+    # Build a mapping from tensor name to filepath
+    tensor_name_to_file = {}
+    for safetensor_file in safetensor_files:
+        with safe_open(safetensor_file, framework="pt", device="cpu") as src:
+            for key in src.keys():
+                tensor_name_to_file[key] = safetensor_file
+
+    def get_tensor(name):
```

Ruff (0.14.10)

Missing return type annotation for private function `get_tensor`

(ANN202)